gpkit/tests/run_tests.py

```diff
@@ -1,53 +1,56 @@
 import unittest
-tests = []
+_TESTS = []
 
-import t_sub
-tests += t_sub.tests
+from .t_sub import _TESTS as t_sub
+_TESTS += t_sub
 
-import t_vars
-tests += t_vars.tests
+from .t_vars import _TESTS as t_vars
+_TESTS += t_vars
 
-import t_nomials
-tests += t_nomials.tests
+from .t_nomials import _TESTS as t_nomials
+_TESTS += t_nomials
 
-import t_constraints
-tests += t_constraints.tests
+from .t_constraints import _TESTS as t_constraints
+_TESTS += t_constraints
 
-import t_posy_array
-tests += t_posy_array.tests
+from .t_posy_array import _TESTS as t_posy_array
+_TESTS += t_posy_array
 
-import t_geometric_program
-tests += t_geometric_program.tests
+from .t_geometric_program import _TESTS as t_geometric_program
+_TESTS += t_geometric_program
 
-import t_gp_solution_array
-tests += t_gp_solution_array.tests
+from .t_gp_solution_array import _TESTS as t_gp_solution_array
+_TESTS += t_gp_solution_array
```

Comment:
I think these should all be `TESTS`, not `_TESTS`. They're not so private/protected if we expect to import them in `run_tests`. 

See also 16719cb29c686154b30169cbf85830f3de4b58e4.